### PR TITLE
Add backend to add topic to PB feature

### PR DESCRIPTION
**Pull request by KristinaDudnyk**

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -24,9 +24,14 @@ const getDataModules = require("./endPoints-file/testEndPoint");
...
+const postTopicToPB = require("./endPoints-file/postTopicToPersonalBoardEndPoint");
+const getlearningTopicsTracker = require("./endPoints-file/getlearningTopicsTrackerEndPoint");
 
 app.get("/modules", getDataModules);
 app.get("/users", getDataUsers);
 app.get("/topics", getTopics);
-
 app.get("/allModulesPage", getDataForAllModulesPage);
+
+app.get("/board", getlearningTopicsTracker);
```
Comment (Abubakar-Meigag) [inline server/index.js:35]: here also please ( getLearningTopicsTrackerEndPoint )

File: server/index.js [hunk 2]
```diff
@@ -24,9 +24,14 @@ const getDataModules = require("./endPoints-file/testEndPoint");
 const getDataUsers = require("./endPoints-file/usersEndPoint");
 const getTopics = require("./endPoints-file/topicsEndpoint");
 const getDataForAllModulesPage = require("./endPoints-file/all_modules_pageEndPoint");
+const postTopicToPB = require("./endPoints-file/postTopicToPersonalBoardEndPoint");
+const getlearningTopicsTracker = require("./endPoints-file/getlearningTopicsTrackerEndPoint");
```
Comment (Abubakar-Meigag) [inline server/index.js:29]: Kristina please can you fix this typo to ( getLearningTopicsTrackerEndPoint )

**Review by Abubakar-Meigag — COMMENTED** [review]
only two comment Kristina thank you

File: server/index.js (hunk 1 above)
Comment (KristinaDudnyk) [reply server/index.js:35]: yep

**Pull request merged**